### ISSUE TITLE
Refresh cdevents.dev look and structure

### ISSUE THREAD

**Issue by afrittoli** [cdevents/cdevents.dev#12]
The project website cdevents.dev has been created by me in a day or two, and it could benefit from an expert refresh, to ensure standard accessibility guidelines are followed, colour palettes are in order, relevant information (community, slack, spec) is easy to discover and finally, that the content is written in a good and clear style.

**Comment by oleg-nenashev** [comment]
Notes from the TOC meeting:

- we have a budget dedicated by the CDF TOC
- we still need someone to work on it

**Comment by afrittoli** [comment]
Things that we need to do on the website:

- user journey in the content
    - most of the content is at the end of the journey today
    - we need to provide more context in the beginning for the reader get the interest in learning more about CDEvents
    - we could use content from the whitepaper
    - expand the content on https://cdevents.dev/ homepage
  
- polished content: 
    - website introduction text (on https://cdevents.dev/ homepage)
    - primer 
    - spec

- diagram content
  - very wordy content today
  - diagram content could help
  - give the same information in multiple ways (text, diagram, code) help the reader to cross-check the content
  - gap analysis to identify where we need diagrams and code

- how do the spec and website content related to each other
  - it can be hard to get complex content displayed on the website and on github
 
- audience strategy: who is the customer for the content
   - today we have a whitepaper
   - need to split the content for different audiences

**Comment by afrittoli** [comment]
This has been implemented.
